apps/data-types.md

```diff
@@ -295,6 +295,7 @@ This object is used for the attributes of basePrice, depositPrice, ecoParticipat
 | productId | string | The unique identifier of the product. |
 | name | string | The name of the product. |
 | shortDescription | string | The short description of the product. |
+| deliveryPeriod | string | The average time of the product being delivered to the customer. |
```

Comment:
we should also mention the unit which is used here. timo suggests the following:
*"By default, "days" is appended as unit. This setting may be changed by the merchant."*